drivers/timer/sam0_rtc_timer.c

```diff
@@ -41,10 +45,13 @@
 
 /*
  * Due to the nature of clock synchronization, reading from or writing to some
- * RTC registers takes approximately six RTC_GCLK cycles. This constant defines
- * a safe threshold for the comparator.
+ * RTC registers takes approximately six RTC_GCLK cycles.
```

Comment:
Update the comment ("six") to match.

Reply:
Do you mean to match the TICK_THRESHOLD?  The actual delay is 6 GCLK + 3 APB cycles.  Generally, APB >> GCLK.  However, if for some reason, they were the same, then the delay would be equivalent to 9 cycles.  I believe the original author assumed APB >> GCLK and "rounded up" to 7 GCLK cycles.

Do you want me to clarify that or change the threshold?

Reply:
/*
 * Due to the nature of clock synchronization, reading from or writing to some
 * RTC registers takes approximately six RTC_GCLK cycles.
 *
 * The actual delay is 6 GCLK + 3 APB cycles. Generally, APB >> GCLK.
 * However, if for some reason, the clocks were the same, then the delay would
 * be equivalent to 9 cycles.
 * 
 * We will assume APB >> GCLK and "round up" to 7 GCLK cycles.
 * The worst case update time after sleep requires 2 sync busy synchronizations.
 * Thus, the threshold is 7 x 2 = 14 cycles.
 *
 * This constant defines a safe threshold for the comparator.
 */